docs: update pr template (#14984)
renovatebot/renovate: .github/pull_request_template.md

<!-- If this is your first pull request: sign the CLA with this GitHub app: https://cla-assistant.io/renovatebot/renovate -->
<!-- Make sure the `Allow edits and access to secrets by maintainers` checkbox is checked on this pull request. -->

> <!-- Please read https://github.com/renovatebot/renovate/blob/main/.github/contributing.md if you don't have yet to reduce noise. -->

Reviewer: ```suggestion
<!-- Please read https://github.com/renovatebot/renovate/blob/main/.github/contributing.md before you create your pull request.-->
```

How about this? It's not clear to readers of the pull request template why reading a file will reduce noise. So it's probably better to tell contributors to read the contributing guide before they start.
